time stamp
int32 npoints
float64 mindist
float64 y1
float64 y2


